test/ufuzz.js

```diff
@@ -575,7 +575,7 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
             createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
       case 28:
         return '(' + createObjectLiteral(recurmax, COMMA_OK, stmtDepth, canThrow) + '[' +
-            createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
+            KEYS[rng(KEYS.length)] + "]) ";
```

Comment:
The more I think about it, mangleProperties is not worth fuzzing. There's way too many problems. You need to test it on actual code designed by humans with well known results.